crates/subspace-runtime/src/lib.rs

```diff
@@ -491,10 +490,6 @@ parameter_types! {
     pub const MaxPendingStakingOperation: u32 = 100;
 }
 
-// `BlockTreePruningDepth` should <= `BlockHashCount` because we need the consensus block hash to verify
-// execution receipt, which is used to construct the node of the block tree.
-const_assert!(BlockTreePruningDepth::get() <= BlockHashCount::get());
```

Comment:
Nice. I noticed earlier that `BlockHashCount` was unused so removing the assertion make sense

Reply:
This is the last use of `static_assertions`, dependency should have been removed as well